test/e2e/network/netpol/network_policy.go

```diff
@@ -799,34 +799,29 @@ var _ = SIGDescribeCopy("Netpol [LinuxOnly]", func() {
 			ValidateOrFail(k8s, model, &TestCase{FromPort: 81, ToPort: 80, Protocol: v1.ProtocolTCP, Reachability: reachabilityAllow})
 		})
 
-		ginkgo.It("should enforce policies to check ingress and egress policies can be controlled independently based on PodSelector [Feature:NetworkPolicy]", func() {
+		ginkgo.It("should keep separate ingress and egress policies for the same target [Feature:NetworkPolicy]", func() {
```

Comment:
to be conservative, i guess  we need to keep this description as is, at least until we migrate the old tests over, in case people are relying on it for a specific ci job or test focus filter.... ?

Reply:
ah that's a good point 👍